### DISCUSSION TITLE
okd 4 support for operators

### DISCUSSION THREAD

**Discussion by rakeshk121** [okd-project/okd#757] [Q&A] [answered]
Does OKD-4 provide full access to users to Operators for installing databases like mongoDB, HDFS, elasitcSearch, janusgraph . Or these operators are supported out of the box only in licensed Openshift Container platform and not in open source OKD?

**Comment by vrutkovs** [comment, marked as answer]
We provide access to operatorhub's community collection of operators. These operators are not officially supported by Red Hat's support and don't require any license.

**Reply by rakeshk121** [reply]
Thanks for the Answer.
